src/cljs/webchange/student_dashboard/toolbar/course_name/views.cljs

```diff
@@ -0,0 +1,30 @@
+(ns webchange.student-dashboard.toolbar.course-name.views
+  (:require
+   [cljs-react-material-ui.reagent :as ui]
+   [re-frame.core :as re-frame]
+   [webchange.subs :as subs]))
+
+(defn- get-styles
```

Comment:
Let's use *.scss files for styling. You can find example in `editor_v2` folder.

Reply:
I have added styles.scss file in the course_name folder & its styles were added with classes property provided by Material UI. Further, can you share the name of the file inside the editor_v2 from which I can take the reference?

Reply:
For example:

- We need to style dialog form in editor_v2. In this case we have to create file `src/cljs/webchange/editor_v2/dialog/dialog_form/styles.scss` 
- Then we need to import it from `src/cljs/webchange/editor_v2/dialog/styles.scss`. 
- Then import dialog styles from editor styles: `src/cljs/webchange/editor_v2/styles.scss`.

And so on.. SCSS files structure should replicate project files structure. For styles the root file is `src/cljs/webchange/ui_framework/styles/index/styles.scss`.